main.js

```diff
@@ -1,41 +1,25 @@
+import BookElement from './modules/BookELement.js';
+import { storeToLocalStorage, getFromLocalStorage } from './modules/LocalStorage.js';
+import { luxon } from './node_modules/luxon/build/global/luxon.js';
+
+const timeIndicator = document.querySelector('.timeIndicator');
+
 const titleInput = document.querySelector('.title');
 const authorInput = document.querySelector('.author');
 const form = document.querySelector('form');
 const booksListElement = document.querySelector('.listOfBooks');
 const navigationLinks = document.querySelectorAll('.nav-link');
 
-class Main {
+class Main extends BookElement {
   constructor() {
+    super();
     this.listOfBooks = [];
   }
```

Comment:
- [ ] Please from the project requirement, you should create the entry point for your JavaScript code called `index.js` :blush: 

- [ ] From the project requirement, you should divide your code into reasonable modules. I think can create a module for this `Main` class in your main.js and import the module into the `main.js` which is supposed to be `index.js` as your app entry point. :relaxed: